src/lang/english.txt

```diff
@@ -1837,7 +1837,6 @@ STR_ABANDON_SCENARIO_QUERY                                      :{YELLOW}Are you
 # Cheat window
 STR_CHEATS                                                      :{WHITE}Cheats
 STR_CHEATS_TOOLTIP                                              :{BLACK}Checkboxes indicate if you have used this cheat before
-STR_CHEATS_WARNING                                              :{BLACK}Warning! You are about to betray your fellow competitors. Keep in mind that such a disgrace will be remembered for eternity
```

Comment:
As `STR_CHEATS_WARNING` is being removed in this PR, would it be great if you had a `Update: Disuse of STR_CHEATS_WARNING` or a similar commit as the string is translated to multiple languages?

Reply:
We have `eints` for that. After this PR is merged, the next time he comes around, he removes the string from allll the other languages. We mere mortals don't have to spend time on that; we outsourced it :D